moj_auth/tests/test_models.py

```diff
@@ -0,0 +1,51 @@
+from django.test import SimpleTestCase
+
+from .utils import generate_tokens
+from ..models import MojUser
+
+
+class UserPermissionsTestCase(SimpleTestCase):
+
+    def setUp(self, *args, **kwargs):
+        super(UserPermissionsTestCase, self).setUp(*args, **kwargs)
+
+        self.user = MojUser(5, generate_tokens(), {
+            'first_name': 'Sam',
+            'last_name': 'Hall',
+            'permissions': [
+                'can_has_cheezburger',
+                'right_to_party',
+                'access_mainframe'
```

Comment:
I might be tempted to rename these to clearer names so it's obvious in the tests below that the permission that's being request is valid/invalid (e.g. `valid_permission_1` and `invalid_permission` or something)?